roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -49,13 +49,16 @@ Messages to and from the availability store.
 ```rust
 enum AvailabilityStoreMessage {
 	/// Query the PoV of a candidate by hash.
-	QueryPoV(Hash, ResponseChannel<PoV>),
+	QueryPoV(Hash, ResponseChannel<AvailableData>),
 	/// Query a specific availability chunk of the candidate's erasure-coding by validator index.
 	/// Returns the chunk and its inclusion proof against the candidate's erasure-root.
 	QueryChunk(Hash, ValidatorIndex, ResponseChannel<AvailabilityChunkAndProof>),
 	/// Store a specific chunk of the candidate's erasure-coding by validator index, with an
 	/// accompanying proof.
 	StoreChunk(Hash, ValidatorIndex, AvailabilityChunkAndProof),
+	/// Store `AvailableData`. If `ValidatorIndex` is provided, also store this validator's 
+	/// `AvailabilityChunkAndProof`.
+	StorePoV(Hash, Option<ValidatorIndex>, u32, AvailableData),
```

Comment:
This should have a result sender and should be called `StoreAvailableData` for consistency